src/plugins/intel_cpu/tests/functional/subgraph_tests/src/subgraph_serialize.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/openvino.hpp"
+#include "openvino/opsets/opset9.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/convolution_params.hpp"
+#include "snippets/op/subgraph.hpp"
+
+using namespace CPUTestUtils;
+using namespace ov::opset9;
+
+namespace SubgraphTestsDefinitions {
+
+class SubgraphSnippetSerializationTest : public ::testing::Test, public CPUTestsBase {};
```

Comment:
General comment on SubgraphSnippetSerializationTest. It looks like the present approach is not scalable. These tests could be generalized to accept any function from src/tests/ngraph_helpers/snippets_ngraph_functions without excessive code repetition. This is important because snippets are extending rapidly and new operations are supported, so we need a scalable mechanism to make sure that we don't break model caching. I don't insist on parametric tests, but it would be nice to reduce effort on testing of new subgraphs.
BTW I agree to merge as-is, but let's create a follow-up ticket and refactor these tests after the release.

Reply:
The ticket has been created: 96916